bundle.Dockerfile

```diff
@@ -28,17 +27,17 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1 \
     operators.operatorframework.io.bundle.channels.v1=alpha \
     operators.operatorframework.io.bundle.channel.default.v1=alpha
 
-# This label gets replaced by render_templates, so it is safer to keep it on its own
-LABEL version="v0.0.0"
-
-# This label maps to the brew build target
-LABEL com.redhat.component="windows-machine-config-operator-bundle-container"
-
-# This last block are standard Red Hat container labels
+# This block are standard Red Hat container labels
 LABEL \
     name="openshift4/windows-machine-config-operator-bundle" \
     License="ASL 2.0" \
     io.k8s.display-name="Windows Machine Config Operator bundle" \
     io.k8s.description="Windows Machine Config Operator's OLM bundle image" \
     summary="Windows Machine Config Operator's OLM bundle image" \
     maintainer="Team Windows Containers <team-winc@redhat.com>"
```

Comment:
since these are almost something like Header labels (with Name, License etc), I'd suggest moving them all the way to the top

Reply:
(still below the FROM directive though)

Reply:
done

Reply:
I am fine with that as long as all the labels are in one spot. i.e
```
FROM...

LABEL..

LABEL...

COPY...

#etc
```